### PR TITLE
12.0 mig product harmonized system

### PR DESCRIPTION
**Pull request by rvalyi**
standard v12 migration

### CODE REVIEW

File: product_harmonized_system/README.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+This module should be usefull for all companies that export physical goods abroad. This module is also used by the Intrastat modules for the European Union, cf the *intrastat_product* module.
+
+
+Installation
+============
+
+This module is NOT compatible with the *report_intrastat* module from the official addons.
+
+
```
Comment (luc-demeyer) [inline product_harmonized_system/README.rst:24]: report_intrastat no longer exists in Odoo 12 OC nor OE. Please adapt.

File: product_harmonized_system/README.rst [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+Usage
+=====
+
+To create H.S. codes, go to the menu *Sales > Configuration > Product Categories and Attributes > H.S. Codes*.
+
```
Comment (luc-demeyer) [inline product_harmonized_system/README.rst:29]: Incorrect for Odoo 12.0.
Also this module doesn't depend on Sales hence no Sales / Configuration menu available.
Please adapt.

File: product_harmonized_system/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    'name': 'Product Harmonized System Codes',
+    'version': '12.0.1.0.0',
+    'category': 'Reporting',
+    'license': 'AGPL-3',
+    'summary': 'Base module for Product Import/Export reports',
+    'author': 'brain-tec AG, Akretion, Noviat, '
+              'Odoo Community Association (OCA)',
+    'depends': ['product'],
+    'conflicts': ['report_intrastat'],
+    'data': [
```
Comment (luc-demeyer) [inline product_harmonized_system/__manifest__.py:19]: report_intrastat -> account_intrastat
conflicts -> excludes

**Review by luc-demeyer — CHANGES_REQUESTED** [review]
cf. inline comments

**Review by alexis-via — APPROVED** [review]

**Comment by alexis-via** [comment]
I'll include Luc's remarks in my next PR to port intrastat_product

**Pull request merged**